wikichron/dash/main.py

```diff
@@ -498,3 +500,31 @@ def update_download_url(slider, query_string, selection_json):
             print(f'Download href updated to: {href}')
 
         return href
+
+
+    @app.callback(
+        Output('user-info', 'children'),
+        [Input('cytoscape', 'tapNodeData')],
+        [State('initial-selection', 'children')]
+    )
+    def update_node_info(userInfo, selection_json):
```

Comment:
Use snake_case instead of camelCase for variable names in Python.

Reply:
oops!!